src/components/v5/common/ActionSidebar/partials/Motions/steps/OutcomeStep/partials/MembersAvatars/MembersAvatars.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from 'react';
+import { Id } from '@colony/colony-js';
+import { useMemberAvatars } from './hooks';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import {
+  calculateLastSliceIndex,
+  calculateRemainingItems,
+} from '~utils/avatars';
+import { notNull } from '~utils/arrays';
+import UserAvatar from '~v5/shared/UserAvatar';
+import { SpinnerLoader } from '~shared/Preloaders';
+import { MembersAvatarsProps } from './types';
+
+const displayName =
+  'v5.common.ActionSidebar.partials.motions.Motion.steps.OutcomeStep.partials.MembersAvatars';
+
+const MembersAvatars: FC<MembersAvatarsProps> = ({
+  currentDomainId = COLONY_TOTAL_BALANCE_DOMAIN_ID,
+  maxAvatars = 4,
+  className,
+}) => {
+  const { loading, watchers } = useMemberAvatars();
```

Comment:
I think this component will be reused in other places later, maybe it will be better to pass `watchers` as a prop?